src/SprykerEco/Shared/Payone/Transfer/payone.transfer.xml

```diff
@@ -518,5 +532,11 @@
         <property name="customerMessage" type="string"/>
         <property name="gender" type="string"/>
     </transfer>
+
+    <transfer name="PayoneKlarnaSessionResponse">
+        <property name="token" type="string"/>
+        <property name="isValid" type="bool"/>
+        <property name="error" type="string"/>
```

Comment:
* Maybe better to use `errorMessage` as in most of the transfer object in this module?

Reply:
++